docs/public/specifications/tests/DNSSEC-TP/dnssec10.md

```diff
@@ -337,7 +337,7 @@ should not be used as of [RFC 5890][RFC 5890#section-2.3.1], section 2.3.1.
 [Method4]:                                    ../Methods.md#method-4-obtain-glue-address-records-from-parent
 [Method5]:                                    ../Methods.md#method-5-obtain-the-name-server-address-records-from-child
 [NOTICE]:                                     ../SeverityLevelDefinitions.md#notice
-[Non-Existent Query Name]:                    #the-non-existent-query-name
+[Non-Existent Query Name]:                    #the-nonexistent-query-name
```

Comment:
This change alone will break the link. In order to avoid that, the corresponding section header should also have its spelling fixed (see line 306) and the links in this document need to be updated accordingly.

Reply:
Thanks. I think I've adjusted this properly.